deepchem/models/torch_models/layers.py

```diff
@@ -47,3 +76,427 @@ def __init__(self, scale, eps=1e-5):
   def forward(self, x):
     norm = self.scale / torch.norm(x, dim=-1, keepdim=True).clamp(min=self.eps)
     return x * norm
+
+
+class Encoder(nn.Module):
```

Comment:
Rename to `MATEncoder` and change documentation to reflect that too.

Reply:
Done!